### ISSUE TITLE
Noise on App Insights

### ISSUE THREAD

**Issue by JFolberth** [Azure/arm-template-whatif#248]
## Describe the noise

**Resource type** (i.e. Microsoft.Storage/storageAccounts)
Microsoft.Insights/componenets
**apiVersion** (i.e. 2020-02-02)

**Client (PowerShell, Azure CLI, or API)**

**Relevant ARM Template code (we only need the resource object for the above `resourceType` and `apiVersion`, but if it's easier you can include the entire template**
```
resource appInsights 'Microsoft.Insights/components@2020-02-02' = {
  name: appInsightsName
  location: location
  kind: 'web'
  tags: tags
  properties:{
    Application_Type:'web'
  }
}
```

**Expected response (i.e. "I expected no noise since the template has not been modified since the resources were deployed)**
No change
**Current (noisy) response (either include a screenshot of the what-if output, or copy/paste the text)**
```
  ~ Microsoft.Insights/components/ai-demo-cus-dev [2020-02-02]
    + properties.Flow_Type:      "Bluefield"
    + properties.Request_Source: "rest"

```

**Additional context**

**Comment by torreymicrosoft** [comment]
Thanks for your patience, and sorry for the long wait on this one. It was recently picked up as one of the older open issues for the ARM Deployments team to review.

This is the same `Microsoft.Insights/components` noise that #305 is tracking: `properties.Flow_Type` ("Bluefield") and `properties.Request_Source` ("rest") reported as additions on every run even though nothing in the template changed. From what I can tell those two are populated on the service side when the component is created, so they come back on the GET that what-if compares your template against and there is nothing in the template for them to match.

I am consolidating the App Insights reports so the property list, the api-versions and the reporters all sit on one thread rather than three.

Duplicate of #305.

If you are seeing noise on `Microsoft.Insights/components` beyond those two properties, please reopen here with the what-if output and I will pick it up separately.
